src/main/resources/db/postgresql/V1_44__2195_Multiple_Contributions_Add_Somu.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS somu_type
+(
+  id                BIGSERIAL   PRIMARY KEY,
+  uuid              UUID        NOT NULL,
+  case_type         TEXT        NOT NULL,
+  type              VARCHAR(20) NOT NULL,
+  schema            JSONB       NOT NULL DEFAULT '{}',
+  active            BOOLEAN     NOT NULL DEFAULT TRUE,
+  
+  CONSTRAINT somu_type_uuid_unique UNIQUE (uuid),
+  CONSTRAINT somu_type_type_unique UNIQUE (type),
```

Comment:
This need removing, we no longer have a unique type as this would stop multiple case_types using the same word for type.

```suggestion
```